astroquery/neos/tests/test_neows.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from ...neos import Neows
+
+from astropy.table import Table
+
+
+def test_observations_from_spk_id():
```

Comment:
These are remote-data tests, right?  They need to be tagged with `@remote_data` if so.